templates/macros.html

```diff
@@ -2,6 +2,14 @@
     Makes the appropriate JS imports for highlighting
     * `languages` An array of strings where each is a valid highlight.js language
 #}
+{% macro active_link(expected, href, text) %}
+    <li class="pure-menu-item">
+        <a class="pure-menu-link {% if active_tab == expected %} pure-menu-active {% endif %}" href="{{ href }}">
+        {{ text }}
+        </a>
+    </li>
+{% endmacro active_link %}
+
```

Comment:
You put this between the `highlight_js` macro and its documentation, could you move it below?

Reply:
Oops, that shows you how bad I am at reading documentation 😆